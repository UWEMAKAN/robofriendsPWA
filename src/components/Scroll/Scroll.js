import React from 'react';

import classes from './Scroll.module.css';

const scroll = (props) => {
  return (
    <div className={ classes.Scroll }>
      { props.children }
    </div>
  );
}

export default scroll;