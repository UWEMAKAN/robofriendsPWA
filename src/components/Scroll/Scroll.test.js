import React from 'react';
import { shallow } from 'enzyme';
import Scroll from './Scroll';
import {createSerializer} from 'enzyme-to-json';

expect.addSnapshotSerializer(createSerializer({mode: 'deep'}));

describe('Scroll Component', () => {
  it('expect to render the Scroll Component', () => {
    expect(shallow(<Scroll />)).toMatchSnapshot();
  });
});