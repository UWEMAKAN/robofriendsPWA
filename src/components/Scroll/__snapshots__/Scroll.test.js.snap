// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scroll Component expect to render the Scroll Component 1`] = `
<div
  className="Scroll"
/>
`;
